frontend/tests/controllers/ContestUsersTest.php

```diff
@@ -551,6 +551,92 @@ public function testContestPracticeWhenOriginalContestHasNotEnded() {
         }
     }
 
+    public function testNextRegisteredContestForUser() {
+        // Create 2 users
+        ['user' => $user1, 'identity' => $identity1] = \OmegaUp\Test\Factories\User::createUser(new \OmegaUp\Test\Factories\UserParams([
+            'username' => 'test_contest_user_1',
+        ]));
+        ['user' => $user2, 'identity' => $identity2] = \OmegaUp\Test\Factories\User::createUser(new \OmegaUp\Test\Factories\UserParams([
+            'username' => 'test_contest_user_2',
+        ]));
+
+        // Get the next registered contest for a user who is not registered in any contest
+        $nextRegisteredContestForUser1 = \OmegaUp\Controllers\Contest::getNextRegisteredContestForUser(
+            $identity1
+        );
+        $this->assertEmpty($nextRegisteredContestForUser1);
+
+        $timePast =  new \OmegaUp\Timestamp(\OmegaUp\Time::get() - 120 * 60);
+        $timeFuture1 =  new \OmegaUp\Timestamp(\OmegaUp\Time::get() + 60 * 60);
+        $timeFuture2 =  new \OmegaUp\Timestamp(\OmegaUp\Time::get() + 120 * 60);
+        // Get 2 active contests
+        $contest1 = \OmegaUp\Test\Factories\Contest::createContest(
+            new \OmegaUp\Test\Factories\ContestParams([
+                'startTime' => $timePast,
+                'finishTime' => $timeFuture2,
+            ])
+        );
```

Comment:
Veo que la prueba está bien elaborada, se valida que el concurso que al que se registró al usuario se obtenga en la función, sólo veo que se puede mejorar un poco, en lugar de hacer las aseveraciones con los tiempos, lo podemos hacer directamente con el concurso, puedes agregar opcionalmente el título a los 3 concursos de la siguiente forma: 
```suggestion
        $contest1 = \OmegaUp\Test\Factories\Contest::createContest(
            new \OmegaUp\Test\Factories\ContestParams([
                'title' => 'Contest_1',
                'startTime' => $timePast,
                'finishTime' => $timeFuture2,
            ])
        );
```